### ISSUE TITLE
Weight gets stuck to 105-110 lb after holding the boombox

### ISSUE THREAD

**Issue by mikarapace** [Koder-Teh/Boombox-Controller#57]

**Comment by BeanM023** [comment]
I had the same thing, think new update broke something.

**Comment by EliwsEverywhere** [comment]
New update definitely broke the weight, same issue for me

**Comment by looklocke** [comment]
It looks like the most recent update fixed a glitch involving negative weight. This mod attempts to set the boombox's weight to 0. I think the new update makes this do funky things???

According to the source posted on thunderstore, the only line of code effecting weight is this:
((GrabbableObject)__instance).itemProperties.weight = 0f;
I bet it could be effectively fixed by changing it to
((GrabbableObject)__instance).itemProperties.weight = 0.1f;
Making the boombox weigh a fraction of the weight instead of 0. I could just be dumb and wrong though.
I can't test because I don't know how to edit the mod.

**Comment by Precisi** [comment]
Yes, another mod had the same issue and they updated it and the items no longer weigh 100lbs each.

**Comment by byxea** [comment]
Still happening on v61, dropping the boombox sets the player's weight to about 100 lbs then picking it back up sets it back to normal, only dying seems to reset the weight.
